### ISSUE TITLE
Register SED-ML URNs as namespace with identifiers.org

### ISSUE THREAD

**Issue by jonrkarr** [SED-ML/sed-ml#186]
https://registry.identifiers.org/prefixregistrationrequest

**Comment by luciansmith** [comment]
I can do this, and plan to once we submit to JIB--it's a co.mbine.org account thing, and I do indeed have one.

**Comment by jonrkarr** [comment]
This just requires filling out a webform. No account is needed.

Typically identifiers resolve to unique webpages. Hopefully they'll accept a namespace where every identifier points to the same page (http://sed-ml.org/urns.html).

**Comment by matthiaskoenig** [comment]
I see two different solutions:
[1] Make this work with current urns page:
The most important thing would be for us to create some pattern which works with URLS. E.g.we just use the URL escape characters, so that the identifiers transform to:
```
urn:sedml:language:bngl -> 
https://identifers.org/sedml-urn/urn%3Asedml%3Alanguage%3Abngl
```
We would then register the pattern: `urn%3Asedml%3A\b(language|format|symbol)\b|%3A.+.` and probably create the subset of pages corresponding to the urns to have a resolvable weblink.

[2] We create an ontology or use a branch of KISAO for this.
This would have the advantage of a simple pattern and we could reuse the KISAO infrastructure. In addition the terms would be explained.

**Comment by jonrkarr** [comment]
My preference is to use EDAM URIs in place of SED-ML URNs. See #94. EDAM is an ontology of formats associated with bioinformatics. As an ontology, it captures much more than the URN strings, and captures this information in a structured way. The ontology already had some of our formats, and I've contributed more. I'm waiting on the developers to approve several more terms.

I think EDAM URIs would also be a good solution to indicating formats in manifests of COMBINE archives for formats beyond the COMBINE standards.

**Comment by fbergmann** [comment]
I'm happy with using EDAM as well.

**Comment by matthiaskoenig** [comment]
Yes, EDAM is a perfect solution for the formats and languages. For the `symbol` time we would use KISAO.

**Comment by jonrkarr** [comment]
I submitted Identifiers.org prefix requests for `sedml.language` and `sedml.format`.

**Comment by fbergmann** [comment]
@jonrkarr  would you still consider edam identifiers valid? what form should they take?

**Comment by jonrkarr** [comment]
I requested the prefixes as part of requesting several prefixes to facilitate validating URIs. I included SED-ML URNs since I was already requesting several other prefixes.

I believe the status of SED-ML is that EDAM identifiers are not (yet) valid. EDAM URIs aren't quite ready for use because there are several proposed terms that still haven't been released.

Going forward (v2), I suggest we used EDAM URIs in place of SED-ML URNs. These EDAM URIs could also be used with OMEX manifests (for formats that are neither official internet media types or official COMBINE formats). 

If we used the identifiers.org form, the URIs would look like http://identifiers.org/edam:format_2585 (SBML). The corresponding official URI would be http://edamontology.org/format_2585. I suggest we use URIs to align with OMEX manifests, rather than using shorthand (e.g., `format_2585`) similar to the SED-ML KiSAO shorthand. 

One more issue is what to do about URIs for COMBINE formats. I think it makes to align SED-ML format URIs with format URIs in OMEX manifests. One option is to encourage everything to use EDAM and also discourage use of the `combine.specifications` namespace.
